my_set/main/models.py

```diff
@@ -1,3 +1,48 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Technology(models.Model):
+    name = models.CharField(max_length=50)
+    def __str__(self):
+        return self.name
+    @property
+    def num_projects(self):
+        return self.project_set.count()
+
+class Industry(models.Model):
+    name = models.CharField(max_length=50)
+    def __str__(self):
+        return self.name
+    @property
+    def num_projects(self):
+        return self.project_set.count()
+    
+class Sets(models.Model):
```

Comment:
what is the purpose of this model, if you have Sets?

Reply:
I wanted to use it, but apparently I used a different method